wrapper/src/main/java/software/amazon/jdbc/Driver.java

```diff
@@ -113,6 +120,15 @@ public static void register() throws SQLException {
     final Driver driver = new Driver();
     DriverManager.registerDriver(driver);
     registeredDriver = driver;
+
+    storageService.registerItemCategoryIfAbsent(
```

Comment:
Should topology be registered here or by HostList provider?

Reply:
I would prefer to register all default item categories here so that we know they have all been registered and don't need to worry about if they've been registered later. If we do not register here then it is less clear whether an item category has been registered or not, so we will need to call the registerIfAbsent method more often, which would be cumbersome since it requires 7 parameters. Alternatively, we could lazily register default item categories automatically so that no one has to worry whether the default categories have been registered or not.

Reply:
Hmm not sure I see these changes. Can you double check?

Reply:
Forgot to push, updated now